tests/integration/test_integration.py

```diff
@@ -83,3 +83,10 @@ async def test_build_and_deploy(ops_test: OpsTest):
     prometheus_k8s_unit = ops_test.model.units[f"{PROMETHEUS_APPLICATION_NAME}/0"]
     prometheus_k8s_private_address = prometheus_k8s_unit.data["private-address"]
     validate_scrape_target_is_added_to_prometheus(prometheus_k8s_private_address)
+
+
+
+async def test_remove_relation(ops_test: OpsTest):
+    await ops_test.model.applications["prometheus-edge-hub"].remove_relation("metrics-endpoint","prometheus-k8s")
+    await ops_test.model.wait_for_idle(apps=["prometheus-edge-hub"], status="blocked", timeout=1000)
```

Comment:
It turns out that the current code will not place the charm in Blocked status so this integration would fail. In this case, please change it back to await for the status to be "Active". Sorry for the confusion.

Reply:
Done have changed it back to 
await ops_test.model.wait_for_idle(apps=["prometheus-edge-hub"], status="active", timeout=1000)

please check